README.md

```diff
@@ -62,7 +62,7 @@ Example: `npx env-cmd cypress run --browser chrome --headed --spec "cypress/e2e/
 
 #### Data Test ID Selectors
 
-The `data-test-id={}` attribute has been added so we can create a more robust and predictable way to select elements for testing purposes, without relying on specific class names or other properties that may be subject to change. Additionally, defining constants for `data-test-id` values makes the test code more readable and maintainable.
+The `data-testid={}` attribute has been added so we can create a more robust and predictable way to select elements for testing purposes, without relying on specific class names or other properties that may be subject to change. Additionally, defining constants for `data-testid` values makes the test code more readable and maintainable.
```

Comment:
I changed the config for `data-test-id` attribute to `data-testid` as I spotted this is already being used elsewhere for other tests so makes sense to follow the same attribute pattern for e2e tests.